src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -1059,7 +1040,7 @@ void GLHEVert::setupTimeVectors()
     this->myRespFactors->GFNC.dimension(tempLNTTS.size(), 0.0);
 
     int index = 1;
-    for (auto &thisLNTTS : tempLNTTS) {
+    for (auto const &thisLNTTS : tempLNTTS) {
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:1062]:(style),[constVariable],Variable 'thisLNTTS' can be declared as reference to const